src/System.Memory/tests/TestHelpers.cs

```diff
@@ -436,6 +464,33 @@ public static Memory<T> DangerousCreateMemory<T>(object obj, int offset, int len
             { new string[] { null, null, null }, new string[] { null, null }, 0},
         };
 
+        public static TheoryData<string[], string[], int> IndexOfAnyNullSequenceData => new TheoryData<string[], string[], int>()
+        {
+            { new string[] { "1", null, "2" }, new string[] { "1", null, "2" }, 0},
+            { new string[] { "1", null, "2" }, new string[] { null, null }, 1},
+
+            { new string[] { "1", null, "2" }, new string[] { "3", null }, 1},
+            { new string[] { "1", null, "2" }, new string[] { "1", "2" }, 0},
+            { new string[] { "1", null, "2" }, new string[] { "3", "4" }, -1},
+
+            { new string[] { null, null, "2" }, new string[] { "3", null }, 0},
+            { new string[] { null, null, "2" }, new string[] { null, "1" }, 0},
+            { new string[] { null, null, "2" }, new string[] { null, "1" }, 0},
+
+            { new string[] { "1", "3", "2" }, new string[] { "1", null, "2" }, 0},
+            { new string[] { "1", "3", "2" }, new string[] { null, null }, -1},
+
+            { new string[] { "1", "3", "2" }, new string[] { null, "1" }, 0},
+
+            { null, new string[] { "1", null, "2" }, -1},
+
+            { new string[] { "1", null, null }, new string[] { null, null, "2" }, 1},
+            { new string[] { null, null, null }, new string[] { null, null }, 0},
+
+            { new string[] { "1", "3", "2" }, null, 0},
```

Comment:
https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F38625~2Fmerge/test~2Ffunctional~2Fcli~2Finnerloop~2F/20190625.69/workItem/System.Memory.Tests

```text
IndexOfAnyNullSequence_String(spanInput: [\"1\", \"3\", \"2\"], searchInput: null, expected: 0)
```

```text
Unhandled Exception of Type Xunit.Sdk.EqualException
Message :
Assert.Equal() Failure
Expected: 0
Actual:   -1
Stack Trace :
   at System.SpanTests.ReadOnlySpanTests.IndexOfAnyNullSequence_String(String[] spanInput, String[] searchInput, Int32 expected) in /_/src/System.Memory/tests/ReadOnlySpan/IndexOfAny.T.cs:line 972

```

Are we sure we should expect 0?

cc @benaadams

Reply:
Did previously; shouldn't now https://github.com/dotnet/corefx/issues/32334

i.e. it should be -1